Add the release note for Nodejs client 1.9.0 (#637)
apache/pulsar-site: release-notes/versioned/pulsar-client-node-1.9.0.md

---
id: pulsar-client-node-1.9.0
title: Pulsar Client Node 1.9.0
sidebar_label: Pulsar Client Node 1.9.0
---

## What's Changed

> * [upgrade] Bump json5 and tsconfig-paths @dependabot in https://github.com/apache/pulsar-client-node/pull/270

Reviewer: ```suggestion
* [upgrade] Bump JSON5 and tsconfig-paths @dependabot in https://github.com/apache/pulsar-client-node/pull/270
```

> * [improve] Add schema typing to consumer and producer @romainbrancourt in https://github.com/apache/pulsar-client-node/pull/278

Reviewer: ```suggestion
* [improve] Add schema type to consumer and producer @romainbrancourt in https://github.com/apache/pulsar-client-node/pull/278
```

* [refactor] optimize ci and build from source code. @shibd in https://github.com/apache/pulsar-client-node/pull/280

> * [fix] where the latest client does not work on Rocky Linux 8 @massakam in https://github.com/apache/pulsar-client-node/pull/285

Reviewer: ```suggestion
* [fix] Fix the issue of the latest client does not work on Rocky Linux 8 @massakam in https://github.com/apache/pulsar-client-node/pull/285
```

* [fix] Rename binary module_name from Pulsar to pulsar @hrsakai in https://github.com/apache/pulsar-client-node/pull/290

> * [upgrade] Upgrade cpp client version to 3.1.1 @shibd in https://github.com/apache/pulsar-client-node/pull/292

Reviewer: ```suggestion
* [upgrade] Upgrade CPP client version to 3.1.1 @shibd in https://github.com/apache/pulsar-client-node/pull/292
```

* [upgrade] Upgrade ansi-regex @hrsakai in https://github.com/apache/pulsar-client-node/pull/293
* [fix] Remove unnecessary / at the end of CPP_CLIENT_BASE_URL @massakam in https://github.com/apache/pulsar-client-node/pull/294
* [fix] Fix the binary host url @RobertIndie in https://github.com/apache/pulsar-client-node/pull/295
* [fix] Make it compatible with GLIBCXX_3.4.19 (CentOS 7) @BewareMyPower in https://github.com/apache/pulsar-client-node/pull/288
* [upgrade] Upgrade cpp client version to 3.1.2 @shibd in https://github.com/apache/pulsar-client-node/pull/297
* [fix] Use the certificate provided by the node.js @shibd in https://github.com/apache/pulsar-client-node/pull/301
* [fix] Fix error when falling back to local build @massakam in https://github.com/apache/pulsar-client-node/pull/302

> * [fix] not compatible with alpine 3.15 env @shibd https://github.com/apache/pulsar-client-node/pull/304

Reviewer: ```suggestion
* [fix] Fix the issue of incompatible with alpine 3.15 env @shibd https://github.com/apache/pulsar-client-node/pull/304
```

> * [fix] Wrap c++ exception when creating client. @shibd in https://github.com/apache/pulsar-client-node/pull/307

Reviewer: ```suggestion
* [fix] Wrap C++ exception when creating client. @shibd in https://github.com/apache/pulsar-client-node/pull/307
```
